File: python/program/Velg.py
class Velg:
    def __init__(self, merk, nama, warna, ukuran):
        self.__merk = merk
        self.__nama = nama
        self.__warna = warna
        self.__ukuran = ukuran
    
    def set_merk(self, merk):
        self.__merk = merk
    
    def set_nama(self, nama):
        self.__nama = nama
    
    def set_warna(self, warna):
        self.__warna = warna
    
    def set_ukuran(self, ukuran):
        self.__ukuran = ukuran
    
    def get_merk(self):
        return self.__merk
    
    def get_nama(self):
        return self.__nama
    
    def get_warna(self):
        return self.__warna
    
    def get_ukuran(self):
        return self.__ukuran

File: Javaaa/program/Velg.java
class Velg {
    private String merk;
    private String nama;
    private String warna;
    private int ukuran;

    public Velg() {
    }

    public Velg(String merk, String nama, String warna, int ukuran) {
        this.merk = merk;
        this.nama = nama;
        this.warna = warna;
        this.ukuran = ukuran;
    }

    public void setMerk(String merk) {
        this.merk = merk;
    }

    public void setNama(String nama) {
        this.nama = nama;
    }

    public void setWarna(String warna) {
        this.warna = warna;
    }

    public void setUkuran(int ukuran) {
        this.ukuran = ukuran;
    }

    public String getMerk() {
        return this.merk;
    }

    public String getNama() {
        return this.nama;
    }

    public String getWarna() {
        return this.warna;
    }

    public int getUkuran() {
        return this.ukuran;
    }
}